src/components/Column/ColumnBody.test.tsx

```diff
@@ -0,0 +1,107 @@
+import { render, screen, within, fireEvent, act } from "@testing-library/react";
+import { mockGetComputedStyle, mockDndSpacing } from "react-beautiful-dnd-test-utils";
+
+import Ticket from "../../interfaces/Ticket";
+import { ColumnBody, DraggableColumnBody } from "./ColumnBody";
+
+describe("<ColumnBody />", () => {
+  test("passed appropriate props - It should render given ticket with title", () => {
+    // Arrange
+    const firstTicketTitle = mockedTickets[0].name;
+    const secondTicketTitle = mockedTickets[1].name;
+    const thirdTicketTitle = mockedTickets[2].name;
+    const mockedOnEditCallback = jest.fn();
+
+    render(<ColumnBody tickets={mockedTickets} onEditClick={mockedOnEditCallback} />);
+
+    // Act
+    screen.getAllByTestId("ticketEditButton").forEach((element) => {
+      fireEvent.click(element);
+    });
+
+    // Assert
+    expect(screen.getByText(firstTicketTitle)).toBeTruthy();
+    expect(screen.getByText(secondTicketTitle)).toBeTruthy();
+    expect(screen.getByText(thirdTicketTitle)).toBeTruthy();
+    expect(mockedOnEditCallback).toBeCalledTimes(mockedTickets.length);
+  });
+});
+
+describe("<DraggableColumnBody />", () => {
+  beforeEach(() => {
+    mockGetComputedStyle();
+  });
+
+  test("passed appropriate props - It should render given ticket with title", async () => {
```

Comment:
@kei95 Is it same test with line 8 in this file?

Reply:
Whoops, it's simply coming from my laziness. Will fix!